spec/lib/vre/ch31_form_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'vre/ch31_form'
+
+RSpec.describe VRE::Ch31Form do
+  let(:claim) { create(:veteran_readiness_employment_claim) }
+  let(:user) { FactoryBot.create(:evss_user, :loa3) }
+  let(:service) { VRE::Ch31Form.new(user, claim) }
+  let(:new_address_hash) do
+    {
+      newAddress: {
+        isForeign: false,
+        isMilitary: nil,
+        countryName: 'USA',
+        addressLine1: '1019 Robin Cir',
+        addressLine2: nil,
+        addressLine3: nil,
+        city: 'Arroyo Grande',
+        province: 'CA',
+        internationalPostalCode: '93420'
+      }
+    }
+  end
+
+  describe '#submit' do
+    describe '#send_to_vre' do
+      let(:faraday_response) { double('faraday_connection') }
+
+      context 'successful submission' do
```

Comment:
```suggestion
      context 'with a successful submission' do
```

Reply:
When describing a context, start description with "with, when," or "without"

Reply:
Also I'd suggest wrapping this in a context of a "valid submission payload" in this case - it seems like you're intending to test that a successful execution of the service method that submits data to VRE